pkg/sources/git/git.go

```diff
@@ -319,6 +321,43 @@ type cloneParams struct {
 	clonePath string
 }
 
+//Defines the interface for returning a single PID value given an executable name
+type PID interface {
+	getPIDs(ctx context.Context, executable string) (int, error)
+}
+
+var ExecutableName = "trufflehog"
```

Comment:
I think this is true in most cases, but not always. Could we use `os.Args[0]` instead?